.github/workflows/rust_ci.yml

```diff
@@ -31,6 +31,7 @@ jobs:
       - name: Cache Rust
         uses: Swatinem/rust-cache@v2
         with:
+          save-if: false
```

Comment:
This was causing the "base" keyed cache to not get updated since this would finish before the other job, so the lint job just won't save.